underlay/build.gradle

```diff
@@ -25,6 +25,7 @@ dependencies {
 
     implementation "org.apache.commons:commons-text:${gradle.vApacheCommonsText}"
     testFixturesImplementation "org.slf4j:slf4j-api:${gradle.vSlf4jApi}"
+    implementation 'org.slf4j:slf4j-simple:2.0.16'
```

Comment:
Is this the same version as the one in the variable in previous line

Reply:
No, different.